extract_verbs.sh

```diff
@@ -3,10 +3,14 @@
 set -e
 
 cd ..
-# Download latest Italian Wikipedia dump
-wget http://download.wikimedia.org/itwiki/latest/itwiki-latest-pages-articles.xml.bz2
+# Download latest Wikipedia dump in language of Chioce
+echo -n "Enter wikipedia dump URL e.g.  http://download.wikimedia.org/itwiki/latest/itwiki-latest-pages-articles.xml.bz2> "
```

Comment:
This should be replaced by a command line argument.
The script should not be interactive.